setup.py

```diff
@@ -158,6 +158,17 @@ def run(self):
         with open(os.path.join(extra_requirements_dir, extra_requirements_filename)) as extra_requirements_file:
             extra_requirements[filename_match.group(1)] = extra_requirements_file.read().splitlines()
 
+try:
+    with open('README.rst', 'r') as readme_file:
+        longdesc = readme_file.read()
+except (IOError, OSError):
```

Comment:
@abadger are there any real cases where README is stripped off the distribution?

Reply:
Oops,  didn't se this before merging.  I don't see a reason.  that we need to check for README.rst to exist.  @StephenWeber  do you remember if there was a reason you had that in there?

Reply:
There wasn't a reason, it probably came along from the example sources I referenced for this kind of work - there is probably some "safety first" reasoning but I can't imagine this failing (or necessarily wanting it to succeed if the file's not present).